phase2/ignition/vanilla/kubelet.service

```diff
@@ -15,6 +16,7 @@ ExecStart=/usr/bin/docker run \
         -v /var/run:/var/run:rw \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet/:/var/lib/kubelet:shared \
+        -v /var/log:/var/log:shared \
```

Comment:
Can you go ahead and change this to `/var/log/containers:/var/log/containers:rw` so that we're [consistent with `kube-deploy`](https://github.com/kubernetes/kube-deploy/blob/10b8f8c3f2d1a5f8c677f2aafcd42ea949144197/docker-multinode/common.sh#L87)?